code/modules/urist/modules/newtrading/NPC/NPC_types/colonist.dm

```diff
@@ -115,6 +115,12 @@
 		speak += "I saw a mudcrab the other day. Vile creatures they are." //the chances of this being heard are so small, but I had to put it in.
 	..()
 
+/mob/living/simple_animal/hostile/npc/colonist/Initialize()
+	. = ..()
+	var/area/A = get_area(src)
+	if(A?.type == /area/spacestations/nanotrasenspace)
```

Comment:
I really don't like this specific area check. That aside, I think this can be addressed in other ways along with some other NPC hostility issues. I'd say revert this chunk here for now, and I'll merge the PR and we can deal with it in the future.